### ISSUE TITLE
Changing vehicle mod turns on engine

### ISSUE THREAD

**Issue by LightSrc** [multitheftauto/mtasa-blue#1633]
**Describe the bug**
Changing vehicle mod automatically turns on vehicle engine. 

**To reproduce**
Bind on key or command function which changes vehicle mod (dff and txd file) and be in the vehicle when use key or command.
Also outputDebugString(getVehicleEngineState(vehicle)) shows false even engine is litterally on and you can drive only forward.

**Expected behaviour**
Before when changed vehicle mod player was warped out of vehicle and vehicle engine state was not changed.

**Version**
1.5.7

**Comment by Zangomangu** [comment]
Related PR #1433 
If you need the old behavior, use setGlitchEnabled("kickoutofvehicle_onmodelreplace", true)

I think the engine turning on is expected behavior, but getVehicleEngineState showing false is a bug.

**Comment by Unde-R** [comment]
I tried to reproduce it, but I don't get false if the engine is on

**Comment by qaisjp** [comment]
@Lightins please can you attach a resource (zip file) with a resource that can reproduce this? thanks!

**Comment by ghost** [comment]
I'm unable to reproduce this bug. Here's the test resource: [enginestatebug.zip](https://github.com/multitheftauto/mtasa-blue/files/5175610/enginestatebug.zip)

**Comment by LightSrc** [comment]
I tested this on default server and engine status showing correctly. Probably when I wrote this issue I was checking engine status on server side, so yea thats my bad.
The main reason why I was writing this issue was because I didnt expected change for not getting kicked out of vehicle in 1.5.7. Which says 
> to avoid breaking existing scripts.


but it litterally broke my server xd

Thanks @Zangomangu for your solution.
